src/common.jl

```diff
@@ -43,48 +43,53 @@ end
 true_rng(model) = model.rng isa Integer ? MersenneTwister(model.rng) : model.rng
 
 function MLJModelInterface.fit(model::MLJFluxModel,
-                               verbosity::Int,
+                               verbosity,
```

Comment:
Please correct me if I'm wrong, but verbosity should is still `Int`, right?

Reply:
Yes, but you no longer need to explicitly type it. There used to be a type ambiguity that required the type annotation but that is now long gone.